KonyAccelerate/controllers/mobile/frmFeedbackController/frmFeedbackController.js

```diff
@@ -0,0 +1,131 @@
+define({
+
+    //Type your controller code here 
+    selectedImage: "starselected.png",
+    unselectedImage: "starunselected.png",
+    objectServiceL: "KonyAccelerateStorageService",
+    dataModelObject: "opinion_answers",
+    ratingQuestionId: "11",
+    feedbackQuestionId: "12",
+    opinion_id: 18,
+    /**
+     * @function onClickOfRatingPrompt
+     * @scope private
```

Comment:
For the scope of the function, according to the standard, we follow there is no need for mentioning scope separately. Just add "@private" and move that to the last.